templates/default/stunnel.conf.erb

```diff
@@ -63,6 +63,11 @@ debug = <%= node['stunnel']['debug'] %>
 output = <%= node['stunnel']['output'] %>
 <% end -%>
 
+; Enable/disable FIPS mode
+<% unless node['stunnel']['fips'].nil? %>
```

Comment:
To keep with the current code style I'd alter this to `if node['stunnel']['fips']` and change the right side tag to include `-%>` which omits it from the file if the block does not execute. This also means moving the comment above within this block so it disappears from the file entirely.